graph/simple/weighted_directed_test.go

```diff
@@ -23,7 +23,7 @@ var (
 func TestWeightedEdgeOvercounting(t *testing.T) {
 	g := generateDummyGraph()
 
-	if neigh := g.From(Node(Node(2))); len(neigh) != 2 {
```

Comment:
Out of curiosity, what was it `Node(Node(foo))` before and not just `Node(foo)`?

Reply:
‽

Those tests need cleaning up.